packages/foam-core/src/note-graph.ts

```diff
@@ -26,30 +26,36 @@ export interface NoteLinkDefinition {
 
 export class Note {
   public id: ID;
+  public frontmatter: object | null;
```

Comment:
is there a reason to have `null` instead of simply an empty object when there are no properties? dealing with `null` is always a bit of a pain :)

Reply:
Will change it :)